src/wallet/rpcwallet.cpp

```diff
@@ -355,17 +355,20 @@ static UniValue sendtoaddress(const JSONRPCRequest& request)
             "5. subtractfeefromamount  (boolean, optional, default=false) The fee will be deducted from the amount being sent.\n"
             "                             The recipient will receive less bitcoins than you enter in the amount field.\n"
             "6. replaceable            (boolean, optional) Allow this transaction to be replaced by a transaction with higher fees via BIP 125\n"
-            "7. conf_target            (numeric, optional) Confirmation target (in blocks)\n"
+            "                             Defaults to false unless explicit estimate mode is used, in which case it defaults to true\n"
+            "7. conf_target            (numeric, optional) Confirmation target (in blocks), or fee rate (sat/kb) if explicit estimate mode is used\n"
```

Comment:
Fee rates are passed in as BTC/kB (see settxfee for example) and amounts can be passed in as either num or string.

Reply:
Fixed.